src/ButtonGroup.js

```diff
@@ -9,35 +9,53 @@ var ButtonGroup = function(ctx, x, y, buttons) {
 
 ButtonGroup.prototype = Object.create(Phaser.Group.prototype);
 ButtonGroup.prototype.constructor = ButtonGroup;
+
 ButtonGroup.prototype.makeButton = function(ctx, btn) {
+    var name = btn.name || '';
     var x = btn.x || 0;
     var y = btn.y || 0;
     var size = btn.size || 25;
     var anchor = btn.anchor || [0.5, 0.5];
```

Comment:
The reason I included all of these as opposed to just the ones that are necessary is so that we could have control over default values. Otherwise if we are missing a value than the phaser default takes over and in most cases that results in a button that's not visible. By defining our own defaults we can at least see that the button is not configured properly and take a look to see why.